test/fixed_variables.jl

```diff
@@ -0,0 +1,122 @@
+using Complementarity
+using Test
+
+
+@testset "Fixing Variables" begin
+
+
+
+    A = (:a,:b,:c,:d)
+    V = (a = 4, b = 3, c = 10, d = 2)
+
+    #Step 1: Construct each mapping on the entire set A, and fix a variable. 
+    m = MCPModel()
+    @variable(m, x[A])
+    @variable(m,y[A])
+    @mapping(m, F[a = A], x[a]+V[a])
+    @mapping(m, G[a = A], x[a]*y[a]+V[a])
+
+
+    JuMP.fix(x[:b],2,force=true)
```

Comment:
What happens if `force=true` is not passed?  Should this be always passed?

Reply:
```force = true``` is necessary with a variable has bound, otherwise and error is thrown. In this example $x[:b]$ has no bounds so removing it won't cause and issue. I default to ```force=true``` as most of the variables in my work are non-negative.

This is actually not the "correct" solution. If $x[:b]]$ is not fixed the solution is $x[:b] = -4$ and $y[:b] = 1$. This also happens in the current version of Complementarity. In GAMS you can see the marginal value for each value of the variables. With the fixed value $x[:b] = 2$ the marginal value is reported at $6 = 2-(-4)$, indicating an incorrect solution. If you know how to view these values currently it would be great, if not this is on my to-do list.

Reply:
I don't know how to obtain the marginal value directly from the PATH Solver, but it might be possible to calculate it using derivatives.